src/main/java/dto/mapper/UserMapper.java

```diff
@@ -0,0 +1,26 @@
+package dto.mapper;
+
+import com.expensetrackersvc.model.User;
+import dto.user.RoleDto;
+import dto.user.UserDto;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Component;
+
+import java.util.HashSet;
+import java.util.stream.Collectors;
+
+@Component
+public class UserMapper {
+
+    public static UserDto toUserDto(User user){
+        return new UserDto()
+                .setEmail(user.getEmail())
+                .setFirstName(user.getFirstName())
+                .setLastName(user.getLastName())
+                .setMobileNumber(user.getMobileNumber())
+                .setRoles(new HashSet<RoleDto>(user.getRoles()
```

Comment:
user.getRoles() Null pointer exception may be thrown here

Reply:
noted . but we need to manually add data for role table